packages/inter-protocol/test/vaultFactory/test-storage.js

```diff
@@ -7,8 +7,10 @@ import '../../src/vaultFactory/types.js';
 import { assertTopicPathData, subscriptionKey } from '../supports.js';
 import { makeDriverContext, makeManagerDriver } from './driver.js';
 
-/** @typedef {import('./driver.js').DriverContext & {
- * }} Context */
+/**
+ * @typedef {import('./driver.js').DriverContext & {
+ * }} Context
```

Comment:
Can this be simplified? [`DriverContext`](https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/test/vaultFactory/driver.js#L74-L89) is already an object.
```suggestion
 * @typedef {import('./driver.js').DriverContext} Context
```

Reply:
It can but I think we're better off leaving it. This style is to make it easy to add to the context, which is common in tests. I appreciate the YAGNI argument against it but I'm the main author of these tests so I'm comfortable making the call.